specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/Jobs.json

```diff
@@ -1126,7 +1126,7 @@
           "description": "List of container definitions for the Container Apps Job.",
           "type": "array",
           "items": {
-            "$ref": "#/definitions/JobExecutionContainer"
+            "$ref": "./CommonDefinitions.json#/definitions/Container"
```

Comment:
Could you update example for the new property

Reply:
There's no new property. It's the same container. It's just we are handling volumeMounts differently in RP.

Reply:
`VolumeMounts` is a new property to job execution. By example, I am referring to https://github.com/Azure/azure-rest-api-specs/blob/release-app-Microsoft.App-2023-11-02-preview/specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/examples/Job_Start.json#L7C8-L7C18